camera/camera_manager.hh

```diff
@@ -11,26 +10,25 @@
 namespace jet {
 
 struct Calibration {
-    cv::Mat camera_matrix;
-    cv::Mat distortion_coefficients;
+  cv::Mat camera_matrix;
+  cv::Mat distortion_coefficients;
 };
+
 struct Camera {
-    std::string serial_number = "uninitialized";
-    std::string v4l_path;
-    int video_index = 0;
-    Calibration calibration;
+  std::string serial_number;
+  std::string v4l_path;
```

Comment:
I understand wanting to separate the notion of a calibration from that of a camera.  But why not have these  composed into a `calibrated_camera` struct to save yourself from constructing parallel `camera_map_` and `calibration_map_`?